pom.xml

```diff
@@ -307,12 +307,13 @@
                                 <lineRate>100</lineRate>
                                 <branchRate>90</branchRate>
                             </regex>
+                            <regex>
+                                <pattern>io.jsonwebtoken.impl.crypto.ECDSA</pattern>
```

Comment:
100% coverage is required for all new code - please add more tests (the exclusions above are for old/legacy code that isn't yet fully up-to-date)